tensorflow/compiler/xla/service/gpu/BUILD

```diff
@@ -949,6 +951,7 @@ cc_library(
     name = "gemm_thunk",
     srcs = ["gemm_thunk.cc"],
     hdrs = ["gemm_thunk.h"],
+    copts = if_cuda_is_configured(["-DGOOGLE_CUDA=1"]),
```

Comment:
This is bizarre, why do we need this with cublasLT? I highly suspect we don't? Same comment for the build option below.

Reply:
This is needed for the `GOOGLE_CUDA' preproc directives. I left this unchanged for now since it's consistent with the implementation of the convolutions.